### ISSUE TITLE
[Bug]: Doused Torch causes Minecraft to never start

### ISSUE THREAD

**Issue by Catomax262** [stal111/Valhelsia-Structures#183]
### Describe the issue

I can't provide much, just this snippet of code:
`<html><body>

  | Exception in thread "main" java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
-- | --
 

<span class="level level-warning multiline">	... 25 more</span>
</body>
</html>`

### Minecraft Version

1.18.2

### Mods Version

- Forge: 40.0.10
- Valhelsia Structures: 0.1.0
- Valhelsia Core: 0.4.0

### Log

```shell
latest.log: https://gist.github.com/Catomax262/912e894f4e3daf44a6dffb8376bf2c57
debug.log: https://gist.github.com/Catomax262/d8da7a22eb6dd79e51497db55014479b
```

**Comment by Vaelzan** [comment]
Use Forge 40.2.5. Forge made a breaking change with 40.2.6 that breaks the mod.

Duplicate of https://github.com/ValhelsiaTeam/Valhelsia-Structures/issues/181

**Comment by Catomax262** [comment]
> Use Forge 40.2.5. Forge made a breaking change with 40.2.6 that breaks the mod.
> 
> Duplicate of #181

happens even on 40.2.5

**Comment by Catomax262** [comment]
> Use Forge 40.2.5. Forge made a breaking change with 40.2.6 that breaks the mod.
> 
> Duplicate of #181

Bruh, i think you didn't read properly, i was at that time at 40.0.10, not 40.2.6

**Comment by Vaelzan** [comment]
> > Use Forge 40.2.5. Forge made a breaking change with 40.2.6 that breaks the mod.
> > Duplicate of #181
> 
> Bruh, i think you didn't read properly, i was at that time at 40.0.10, not 40.2.6

No, it's you that didn't read properly. Your log included 40.2.10, not 40.0.10. You typed the wrong version.

**Comment by Vaelzan** [comment]
> > Use Forge 40.2.5. Forge made a breaking change with 40.2.6 that breaks the mod.
> > Duplicate of #181
> 
> happens even on 40.2.5

Only because you *also* updated Valhelsia Structures to the version we just released that works around that Forge change.
